recipes/default.rb

```diff
@@ -24,11 +24,20 @@
   action node['threatstack']['pkg_action']
 end
 
+if node['threatstack']['deploy_key'].nil?
+  api_keys = Chef::EncryptedDataBagItem.load(
+    node['threatstack']['data_bag_name'],
+    node['threatstack']['data_bag_item'])
```

Comment:
Having this parenthesis on the same line as the last parameter doesn't seem to match how functions are invoked elsewhere in the codebase. I'd suggest putting the `)` on its own line.